local-process-quickstart.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash cloudshell for a seamless experience.
```

Comment:
I don't expect all users to open the .sh file before running it. Should we have a first step where we ask the user to confirm he has all these dependencies? Else we're just going to let the user run the script, get an error and be unhappy about the whole experience.

Reply:
Based on my conversation with Yuval, the intent of this script is not to build a full-fledged perfect installation script that checks the existence of all pre-requisites. This should be just a getting started quickstart script.
Having said that, we should bubble up these pre-reqs in the documentation for connect too.